README.md

```diff
@@ -20,6 +21,11 @@ $ npm install @frontend/web-app-monitoring
 
 ## Usage in project
 
+### Browser or Server Methods
+
+- `debug, info, warn, error`: adds item to logs (either local or remote, depending on setup);
+- `reportError`: reports error to monitoring system (either local or remote, depending on setup);
+
 ```js
 import { BrowserMonitoringService } from '@frontend/web-app-monitoring';
 
```

Comment:
missed example of browser monitoring initiation, params needed and maybe an example with error boundary. How do you think? 🤔